controllers/reconcile.go

```diff
@@ -140,8 +140,21 @@ func populateContainer(d *apps.Deployment, frontend *crd.Frontend, frontendEnvir
 			},
 		},
 		VolumeMounts: populateContainerVolumeMounts(frontendEnvironment),
-	},
-	}
+		LivenessProbe: &v1.Probe{
+			ProbeHandler: v1.ProbeHandler{
+				HTTPGet: &v1.HTTPGetAction{Path: "/", Port: intstr.FromInt(80)},
+			},
+			InitialDelaySeconds: 60,
```

Comment:
Is there a reason to wait so long? 60s is a long time to wait for the initial delay - The period is also very long meaning it will take us a long time to find out if the pod goes sideways.

Reply:
I've never done this before so I didn't have any idea what to put in. What do you recommend?